qfieldsync/utils/qt_utils.py

```diff
@@ -29,3 +29,18 @@ def selectFolder(line_edit_widget):
 
 def make_folder_selector(widget):
     return partial(selectFolder, line_edit_widget=widget)
+
+
+def set_available_actions(combobox, layer_source):
```

Comment:
that doesn't seem to belong to qt_utils. You can rather create a gui/utils.py ?

Reply:
TBH I had no idea where to put it. The function gets `QComboBox` as a param, similar to `def selectFolder(line_edit_widget: QLineEdit)` in the same file, that's why I thought is a good place to go.

Reply:
Should I really move it to another file, tried it and it feels like one more file to manage, rather than clear separation of the functionality? No strong opinions, just slight feelings.

Reply:
it's just that it contains QFieldSync specific code and could not be reused in another plugin, while the rest of the file yes.
I'd prefer a new file personally, but no biggie.

Reply:
Deal